misk-aws/src/main/kotlin/misk/jobqueue/sqs/ResolvedQueue.kt

```diff
@@ -16,6 +16,9 @@ internal class ResolvedQueue(
   val client: AmazonSQS
 ) {
 
+  val queueName: String
```

Comment:
very optional: I'd be okay dropping this alias; between `queue.queueName` and `queue.name.value`, we're only saving 1 char.